index.html

```diff
@@ -1,110 +1,164 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="utf-8">
+    <meta charset="utf-8" />
+
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0"
+    />
+    <meta name="HandheldFriendly" content="true" />
+    <meta name="mobile-web-app-capable" content="yes" />
+
     <title>FRONTEND CHALLENGE</title>
-    <link href="style.css" rel="stylesheet" type="text/css" />
-    <link href="https://www.fontify.me/wf/7116669ff62ea6ce905b50766eaca6b8" rel="stylesheet" type="text/css">
-    <link href="https://www.fontify.me/wf/e6a3c17c14d6295d075e29aa38e455ce" rel="stylesheet" type="text/css">
-    <link href="https://www.onlinewebfonts.com/fonts/SF_Compact_Display" rel="stylesheet" type="text/css">
-      </head>
+
+    <link href="./assets/styles/reset.css" rel="stylesheet" type="text/css" />
+    <link href="./assets/styles/style.css" rel="stylesheet" type="text/css" />
+
+    <link
+      href="https://www.fontify.me/wf/7116669ff62ea6ce905b50766eaca6b8"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link
+      href="https://www.fontify.me/wf/e6a3c17c14d6295d075e29aa38e455ce"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link
+      href="https://www.onlinewebfonts.com/fonts/SF_Compact_Display"
+      rel="stylesheet"
+      type="text/css"
+    />
+  </head>
   <body>
-    <section class="container">
-      <nav>
-        <div class="base"></div>
-        <div class="backarrow"> </div>
-            <h1> Bus Routes </h1>
+    <!-- Header -->
+    <header>
+      <div class="StatusBar"></div>
```

Comment:
Class names are all more descriptive now, instead of "base" I used "StatusBar" and so forth.